src/utils/transactions/TransactionResultUtil.ts

```diff
@@ -288,15 +282,57 @@ export const EcommerceMaybeInterruptStatusCodeEnumType =
     "EcommerceMaybeInterruptStatusCodeEnumType"
   );
 
+/**
+ * This function will match any status code from closePaymentResultError with any
+ * status code defined in the ClosePaymentErrorsMap item.
+ *
+ * NOTE:
+ * ClosePaymentErrorsMap supports placeholders, so 5xx will match any error >= 500
+ */
+function evaluateClosePaymentResultError(
```

Comment:
add tests into `src/utils/__tests__/TransactionResultUtil.test.ts` to cover this logic

Reply:
done here 57da1bd59e9c6ef583d1de66a65f727b9e029279